README.md

```diff
@@ -42,62 +41,38 @@ dependencies {
 
 ## Usage
 
-The following sections demonstrate how to retain instances in activities and fragments.
-
-For simplicity, all examples will retain the following class:
+The following sections demonstrate how to retain instances in activities and fragments. For simplicity, all examples will retain the following class:
 
 ```kotlin
 data class ViewModel(var counter: Int = 0)
 ```
 
-### Use Retained in an Activity
-
-To retain an instance with an `Activity`, do:
+### Use Retained in Activities and Fragments
```

Comment:
I saw this as too many small sections saying almost the same thing but slightly different. I personally liked it better when I put them all together in the same snippet, this way the content has the same info but it's smaller and has less pauses so it's easier to absorb the whole thing, if that makes sense.